Remove/reword reference to global python environment in Managing Packages (#27)
Ada-Developers-Academy/core-software-tools: managing-packages/packages-and-managing-packages.md

1. How to list all the packages installed on our computer
1. How to install packages

### Installation

During "installfest" we used a tool named `pyenv` to install Python and `pip` along with it.

To verify that `pip` is installed, we can run our first `pip` command in the terminal:

```bash
pip --version
```

And we should see output similar to:

```bash
pip 24.3.1 from /Users/<YOUR_USER_NAME>/.pyenv/versions/3.13.1/lib/python3.13/site-packages/pip (python 3.13)
```

When reading our output we should look out for:

> - If we see a messgae that `pip` is not found. 
>   - If you see this, refer to the `pip` docs for an installation guide.
> - If we see that `.pyenv` is not part of the path to the `pip` installation. 
>   - If you see this, reach out in #study-hall on Slack for assistance.

Reviewer: Started with a small typo, but I also struggled with all the If/If/If/If here. What about rephrasing something like this?

```suggestion
- Do we see a message that `pip` is not found?
  - If we get a not found message, refer to the `pip` docs for an installation guide.
- Do we _not_ see `.pyenv` included in the path to the `pip` installation?
  - If the path doesn't include `.pyenv`, reach out in #study-hall on Slack for assistance.
```

Reviewer: Also, is refering folks to a pip installation guide what we want to do? It should have come with `pyenv`, so if it's missing I'd like to know why, and make sure it's resolved appropriately. Maybe just send them to #study-hall in either situation?

Author: Yeah, that's the original advice, I agree that sending them to #study-hall would likely be preferred if pip is somehow not installed/in their path
